Combined all tasks + edited bonus assignemnt

katharineholmes46 opened the pull request.

angelaradulescu commented:
first hop: 

I can't do "server on" in the MasterTask folder. it's giving me the error below.

did you change any settings that might have caused this? 

Traceback (most recent call last):
  File "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/psiturk/experiment_server.py", line 72, in <module>
    launch()
  File "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/psiturk/experiment_server.py", line 69, in launch
    ExperimentServer().run()
  File "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/app/base.py", line 185, in run
    super(Application, self).run()
  File "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/app/base.py", line 71, in run
    Arbiter(self).run()
  File "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py", line 169, in run
    self.manage_workers()
  File "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py", line 477, in manage_workers
    self.spawn_workers()
  File "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py", line 542, in spawn_workers
    time.sleep(0.1 * random.random())
  File "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py", line 209, in handle_chld
    self.reap_workers()
  File "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py", line 459, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>

katharineholmes46 commented:
i entered the aws and psiturk credentials, but i don't think that should
make any difference. i didn't change any of the other settings.

On Thu, Jun 18, 2015 at 5:06 PM, Angela Radulescu <notifications@github.com>
wrote:

> first hop:
>
> I can't do "server on" in the MasterTask folder. it's giving me the error
> below.
>
> did you change any settings that might have caused this?
>
> Traceback (most recent call last):
> File
> "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/psiturk/experiment_server.py",
> line 72, in
> launch()
> File
> "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/psiturk/experiment_server.py",
> line 69, in launch
> ExperimentServer().run()
> File
> "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/app/base.py",
> line 185, in run
> super(Application, self).run()
> File
> "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/app/base.py",
> line 71, in run
> Arbiter(self).run()
> File
> "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py",
> line 169, in run
> self.manage_workers()
> File
> "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py",
> line 477, in manage_workers
> self.spawn_workers()
> File
> "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py",
> line 542, in spawn_workers
> time.sleep(0.1 * random.random())
> File
> "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py",
> line 209, in handle_chld
> self.reap_workers()
> File
> "/Users/angelar/Library/Enthought/Canopy_64bit/User/lib/python2.7/site-packages/gunicorn/arbiter.py",
> line 459, in reap_workers
> raise HaltServer(reason, self.WORKER_BOOT_ERROR)
> gunicorn.errors.HaltServer:
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angelaradulescu/riskturk/pull/20#issuecomment-113290059>
> .
>

angelaradulescu commented:
ok thanks -- which file did you change?

katharineholmes46 commented:
I just added master task. So task.js was changed and i added some static pages also

Sent from my iPhone

> On Jun 18, 2015, at 17:11, Angela Radulescu <notifications@github.com> wrote:
> 
> ok thanks -- which file did you change?
> 
> —
> Reply to this email directly or view it on GitHub.
>

angelaradulescu commented:
and where did you enter the credentials / can you copy them over? 

I suspect this happened because we have different psiturk config files.

katharineholmes46 commented:
Try this:

[AWS Access]
aws_access_key_id = AKIAJ54URWVZB37RHEBQ
aws_secret_access_key = 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
aws_region = us-east-1

[psiTurk Access]
psiturk_access_key_id = wkYzXYhX7hLsQxYnZkQfgV
psiturk_secret_access_id = PucBufQzobdpAyfzPiQSEf

On Thu, Jun 18, 2015 at 5:20 PM, Angela Radulescu <notifications@github.com>
wrote:

> and where did you enter the credentials / can you copy them over?
>
> I suspect this happened because we have different psiturk config files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angelaradulescu/riskturk/pull/20#issuecomment-113293006>
> .
>
